urdu/jesa-karo-ge

```diff
@@ -0,0 +1,4 @@
+ایک دفعہ ہم ان کے گھر گئے تو وہ سو رہے تھے
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.